recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
```

Comment:
Looks like it is complaining that the terminal is not correctly configured when running `git log`. Seems adding `export TERM=xterm` fixes that issue for me in the docker container.